apps/web/src/styles/styles.ts

```diff
@@ -1,2 +1,11 @@
 export const planningFormBox =
-  'w-full justify-right print:hidden border-2 bg-white rounded p-3 mt-4';
+  'w-full justify-right print:hidden border-2 bg-white rounded p-4 mt-4';
+
+export const occupationItemBox = 'flex items-center h-16 px-4';
```

Comment:
Move all the styles here to globals.css and use the className to refer them in code

Reply:
If this is not used, then it can be deleted!